### PR TITLE
Remove manually maintained msvc projects from src/mono

### PR DESCRIPTION
**Pull request by akoeplinger**
They're no longer needed after https://github.com/dotnet/runtime/pull/44976

### CODE REVIEW

File: src/mono/cmake/config.h.in [hunk 1]
```diff
@@ -1,6 +1,69 @@
+#ifndef __MONO_CONFIG_H__
+#define __MONO_CONFIG_H__
+
 #ifdef _MSC_VER
-#include <msvc/msvc-win32-support.h>
-#include <msvc/msvc-disabled-warnings.h>
+
+// FIXME This is all questionable but the logs are flooded and nothing else is fixing them.
```
Comment (stephentoub) [inline src/mono/cmake/config.h.in:6]: Is there an issue tracking this?

**Review by stephentoub — APPROVED** [review]
This is very pretty:
image

File: src/mono/cmake/config.h.in (hunk 1 above)
Comment (akoeplinger) [reply src/mono/cmake/config.h.in:6]: Hm no, this came from msvc-disabled-warnings.h which has existed for a long time in mono/mono :)

**Review by kg — APPROVED** [review]

File: src/mono/cmake/config.h.in (hunk 1 above)
Comment (lateralusX) [reply src/mono/cmake/config.h.in:6]: warnings are on other platforms as well (not msvc specific), just building with lower warning level or explicitly disabled them as part of build on other platforms.

**Pull request merged**